File: Java/src/World/body/plant/Grass.java
package World.body.plant;

import GUI.Color_obj;
import World.Point;
import World.World;
import World.body.Plant;

import java.io.Serializable;

public class Grass extends Plant implements Serializable {
    private static final long serialVersionUID = 987654321L;
    public Grass(Point point_location, World world) {
        super(point_location, world);
        setName("Grass");
        setPower(0);
        setColor(Color_obj.GRASS);
    }

    @Override
    public int get_stronger() {
        return 0;
    }
}

File: Python/World/Body/Plant/grass.py
from World.Body.plant import Plant
from World.point import Point
from GUI.colors import Colors


class Grass(Plant):
    def __init__(self, point_location, world):
        super().__init__(point_location, world)
        self.set_name("Grass")
        self.set_power(0)
        self.set_color(Colors.GRASS)

    def get_stronger(self):
        return 0
